src/app/plugins/8ball.plugin.ts

```diff
@@ -1,39 +1,44 @@
-import { Plugin } from '../../common/plugin';
-import { IContainer, IMessage, ChannelType } from '../../common/types';
+// the old usage string was obviously '8ball', although it could be anything moving forward
+//
+// when executed, lion should respond with an embed containing a message and an embedded image of an 8 ball
+
 import { MessageEmbed } from 'discord.js';
+import { Plugin } from '../../common/plugin';
+import { ChannelType, IContainer, IMessage } from '../../common/types';
 
-export default class EightBallPlugin extends Plugin {
-  public commandName: string = '8ball';
-  public name: string = '8Ball Plugin';
-  public description: string = 'A magic fortune telling 8Ball.';
-  public usage: string = '8ball';
-  public override pluginAlias = [];
-  public permission: ChannelType = ChannelType.Public;
+const replyOptions = [
+  'No.',
+  "It's not impossible, but I doubt it.",
+  "Don't bet on it.",
+  "It's about as likely as your code working on the first try.",
+  'Yes.',
+  'It is certain.',
+  'You can count on it.',
+  'I personally guarantee it.',
+  'Ask again later.',
+];
```

Comment:
Tbh, a better question is why this is even a class at all. It's literally just a utility function with no state.

Reply:
Good point. I will change this.

Reply:
Having the function defined in the class would seem to fit the project style more.

Reply:
Wait, just realized this thread was in reference to the 8ball plugin and not `RoleService` (which is actually just a utility function). What are the advantages of adding it to the class with `private static readonly`?

Reply:
We've never enforced this in the past for plugins since it doesn't seem to provide much. I don't think we in danger of someone *accidently* accessing `EightBallPlugin.replyOptions`. It wouldn't hurt to add the modifiers but we certainly don't enforce it

Reply:
Since it is not exported it is already effectively private. That's why I'm not sure what benefit we get from putting it in the class.